package.json

```diff
@@ -19,17 +21,17 @@
     "killApp": "(lsof -t -i:7080 | xargs kill)",
     "killStorybook": "(lsof -t -i:9001 | xargs kill)",
     "lighthouse": "lighthouse http://localhost:7080/news/articles/c85pqyj5m2ko --output-path=./reports/report.html --view",
-    "postshrinkwrap": "if test -z $CI; then \n npm run force-package-lock-https \n fi",
+    "postshrinkwrap": "if test -z $CI; then \n npm run force-package-lock-https \n fi; \n git update-index --assume-unchanged .env",
```

Comment:
```suggestion
    "postshrinkwrap": "if test -z $CI; then \n npm run force-package-lock-https \n fi; && git update-index --assume-unchanged .env",
```

Reply:
![image](https://user-images.githubusercontent.com/7791726/52428027-12137880-2af9-11e9-95d8-45d0dd957701.png)

Reply:
It didn't work 🤷‍♂️